spec/entities/configurations/kalibro_configuration_spec.rb

```diff
@@ -17,6 +17,14 @@
 require 'spec_helper'
 
 describe KalibroClient::Entities::Configurations::KalibroConfiguration do
+  describe 'accessors' do
+    it 'should have accessors for the needed attributes' do
```

Comment:
Aren't we just testing the way ruby is supposed to work here?

If this is for documentation purposes (which I think is good), maybe we should consider printing more specific messages.

What do you think of:

```ruby
[:name, :description, :language].each do |attr|
  it "should have accessors for the #{attr} attribute" do
    expect(subject).to have_attr_accessor(attr)
  end
end
```

Caveat: I'm not entirely sure that code snippet works :wine_glass: